src/async_impl/response.rs

```diff
@@ -270,8 +273,35 @@ impl Response {
     /// # Ok(())
     /// # }
     /// ```
-    pub async fn bytes(self) -> crate::Result<Bytes> {
-        hyper::body::to_bytes(self.body).await
+    pub async fn bytes(mut self) -> crate::Result<Bytes> {
```

Comment:
So it looks like you mean for this to only apply to the methods that buffer the full body. It's not meant to apply to the streaming methods? I don't ask because that's wrong (either decision could make sense), but that if that's the case, the docs should mention it. Or it should apply to streaming too. There's pros and cons for both choices.